### ISSUE TITLE
Repeated sentence in angular.dev's Anatomy of components page

### ISSUE THREAD

**Issue by racz16** [angular/angular#53030]
### Describe the problem that you experienced

In angular.dev, in the `In-depth Guides` / `Components` / `Anatomy of components` page, in the beginning, there is a sentence that is repeated: 
> You provide Angular-specific information for a component by adding a [@Component](https://angular.dev/api/core/Component) [decorator](https://www.typescriptlang.org/docs/handbook/decorators.html) on top of the TypeScript class: You provide Angular-specific information for a component by adding a [@Component](https://angular.dev/api/core/Component) [decorator](https://www.typescriptlang.org/docs/handbook/decorators.html) on top of the TypeScript class:

### Enter the URL of the topic with the problem

https://angular.dev/guide/components

### Describe what you want to experience that would fix the problem

I'd like to see the sentence only once.

### Add a screenshot if that helps illustrate the problem

![image](https://github.com/angular/angular/assets/19560859/84f6a67a-1195-435b-8548-eefcdb858df2)

**Comment by aramirezj** [comment]
Seems like it is already fixed by https://github.com/angular/angular/pull/52587

**Comment by JoostK** [comment]
Duplicate of #53029
